website/content/docs/release-notes/nomad/v1_9_x.mdx

```diff
@@ -0,0 +1,109 @@
+  correlate CPU cores with memory nodes and assign tasks to run on specific CPU
+  cores so as to minimize any cross-memory node access patterns. With Nomad 1.9,
+  we are expanding this functionality to also correlate
+  [devices](/nomad/docs/concepts/plugins/devices) to memory nodes and enable
+  NUMA-aware scheduling to take device associativity into account when making
+  scheduling decisions. Refer to [numa
+  Block](/nomad/docs/job-specification/numa) for configuration details.
+
+- **Golden job versions**: This feature introduces a way to preserve and compare
+  historical versions of a job. Prior to this, any change to a job would push
+  older versions into garbage collection. Now, you can tag a version to save it
+  from garbage collection. Additionally, you can compare job versions by tag
+  name and revert by tag name. Refer to [Job
+  versions](/nomad/docs/concepts/job.mdx#job-versions) for details and links to
+  how to use job version tagging.
+
+- **Virt task driver (beta)**: The new Virt task driver uses
+  [libvirt](https://libvirt.org/) to manage virtual machines created by KVM,
+  Xen, VMware ESXi, QEMU, and other virtualization technologies. Using this
+  driver, you can perform the following:
+
+  - Start and stop a virtual machine and run a process on it.
+  - Assign a workload identity to the running task inside the virtual machine.
+  - Pass configuration values from Vault or other services using consul-template.
+  - Mount host directories into virtual machines to procure the allocation and task directories inside them.
+  - Configure port forwarding from the Nomad node to the virtual machine.
+  - Access the running task through the task API.
+
+  Refer to the [Virt driver docs](/nomad/plugins/drivers/virt) for installation and usage guides.
+
+- **Exec2 Task Driver (GA):** The `exec2` driver is used to execute a command
+for a task. It offers a security model optimized for running 'ordinary'
+processes with very low startup times and minimal overhead in terms of CPU,
+disk, and memory utilization. The `exec2` driver leverages kernel features such
+as the Landlock LSM, cgroups v2, and the unshare system utility. Due these
+enhancements, tasks no longer require leveraging filesystem isolation based on
+chroot, which provide an enhanced security profile and improved performance for
+the Nomad operator. Refer to [Exec2 Task Driver](/nomad/plugins/drivers/exec2)
+for more details.
+
+- **Improved IPv6 support**: Nomad has improved support for Nomad's IPv6
+capabilities. We have tested and resolved some known issues with
+server-to-server and server-to-client communications. Additionally we have
+tested the Nomad CLI and UI service integrations with Consul and Vault, workload
+identity, service registration, host networking, bridge networking, and the
+Docker driver. More work will likely need to be done with IPv6 support as we
+explore integrations with Consul service mesh.
+
+
+## What's deprecated
+
+N/A
```

Comment:
Do we want to refer to the deprecations described here https://developer.hashicorp.com/nomad/docs/release-notes/nomad/upcoming#nomad-1-10-0-lts?